data-sources/api/src/commonMain/kotlin/EpisodeSort.kt

```diff
@@ -110,22 +131,35 @@ sealed class EpisodeSort : Comparable<EpisodeSort> {
     }
 }
 
+private fun getSpecialByRaw(raw: String): EpisodeSort {
+    val type = EpisodeType.entries.firstOrNull { entry -> raw.startsWith(entry.value, ignoreCase = true) }
+    if (type == null || raw.length < type.value.length) return EpisodeSort.Unknown(raw)
```

Comment:
```suggestion
    val type = EpisodeType.entries.firstOrNull { entry -> raw.startsWith(entry.value, ignoreCase = true) }
        ?: return EpisodeSort.Unknown(raw)
```

`raw.length < type.value.length` 永远为 `false`